pkgs/development/libraries/qt-5/5.15/default.nix

```
@@ -55,15 +55,15 @@ let
     # See https://lists.qt-project.org/pipermail/interest/2021-March/036387.html
     qtwebengine =
       let
-        branchName = "5.15.3";
-        rev = "a059e7404a6db799f4da0ad696e65ae9c854b4b0";
+        branchName = "5.15.5";
+        rev = "9711f64c5082040cb76f6da5ef4a16037dbda08f";
```

It seems that we can use the tags:

```suggestion
        rev = "v${version}-lts";
```

Can you also drop the comment above? (`Even if developed in the public ...`) as it does not seem valid anymore.